README.md

```diff
@@ -1,71 +1,30 @@
 # code-coverage-api-plugin
-(This plugin is now under development)
 
-This plugin serves as API to integrate and publish multiple coverage report types.
+[![Build Status](https://ci.jenkins.io/job/Plugins/job/code-coverage-api-plugin/job/dev/badge/icon)](https://ci.jenkins.io/job/Plugins/job/code-coverage-api-plugin/job/dev/)
+[![Gitter](https://badges.gitter.im/jenkinsci/code-coverage-api-plugin.svg)](https://gitter.im/jenkinsci/code-coverage-api-plugin?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-## How to use it
-
-Code Coverage API plugin now supports Cobertura and Jacoco.
-
-#### Config maven to generate coverage reports.
 
-For **Cobertura**
-```xml
-<build>
-    <plugins>
-        <plugin>
-            <groupId>org.codehaus.mojo</groupId>
-            <artifactId>cobertura-maven-plugin</artifactId>
-            <version>2.7</version>
-            <configuration>
-                <formats>
-                    <format>xml</format>
-                </formats>
-                <check/>
-            </configuration>
-            <executions>
-                <execution>
-                    <phase>package</phase>
-                    <goals>
-                        <goal>cobertura</goal>
-                    </goals>
-                </execution>
-            </executions>
-        </plugin>
-    </plugins>
-</build>
-```
+This plugin serves as API to integrate and publish multiple coverage report types. 
+More information see [https://jenkins.io/projects/gsoc/2018/code-coverage-api-plugin/](https://jenkins.io/projects/gsoc/2018/code-coverage-api-plugin/).
 
-For **Jacoco**
-```xml
-<build>
-    <plugins>
-        <plugin>
-            <groupId>org.jacoco</groupId>
-            <artifactId>jacoco-maven-plugin</artifactId>
-            <version>0.8.1</version>
-            <executions>
-                <execution>
-                    <goals>
-                        <goal>prepare-agent</goal>
-                    </goals>
-                </execution>
-                <execution>
-                    <id>report</id>
-                    <phase>package</phase>
-                    <goals>
-                        <goal>report</goal>
-                    </goals>
-                </execution>
-            </executions>
-        </plugin>
-    </plugins>
-</build>
-```
+## Features
+- Pipeline support.
+- Modernized coverage chart.
+- Coverage trend support.
+- Multi-report aggregated.
+- Rest API.
+- Auto detect report.
+- Easy to implement.
 
-Then plugin will automatically find reports (Only Support Cobertura now) according to your auto detect path.
-
-Also, we can specify report path for each coverage tool.
+## How to use it
 
-For more details, see the [Introduction Blogpost](https://jenkins.io/blog/2018/06/13/code-coverage-api-plugin/).
+Code Coverage API plugin now supports Cobertura, Jacoco and llvm-cov.
```

Comment:
It is not correct, because implementations are in external plugins. Links should be provided so that the guideline is explicit about that